client/src/app/containers/Layout/index.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { useInjectReducer } from 'redux-injectors';
+import clsx from 'clsx';
+import { Divider, Drawer, IconButton } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
+import { selectLoggedIn } from '../../../auth/selectors';
+import { sliceKey, reducer } from '../../../auth/slice';
+import Alert from '../Alert';
+import Navbar from '../../components/Navbar';
+import LoggedInDrawer from '../LeftDrawer/LoggedInDrawer';
+import LoggedOutDrawer from '../LeftDrawer/LoggedOutDrawer';
```

Comment:
📖  🖊️  What's the standard for import ordering? this is kind of a mess and it's hard to read/understand. Also, I'm starting to like @AaronRMichael 's idea of absolute paths

Reply:
I installed a linter ([`eslint`](https://eslint.org/)) as well as a [plugin](https://github.com/benmosher/eslint-plugin-import) which enforces import ordering and grouping. I set it up according to and set it up according to [this example](https://dev.to/otamnitram/sorting-your-imports-correctly-in-react-213m). Still need to look into absolute paths.